src/commands/synthetics/reporters/junit.ts

```diff
@@ -163,15 +165,15 @@ export class JUnitReporter implements Reporter {
         // It's a browser test.
         for (const stepDetail of result.result.stepDetails) {
           const {browser_error, error, warning} = this.getBrowserTestStep(stepDetail)
-          testCase.browser_error = browser_error
-          testCase.error = error
-          testCase.warning = warning
+          testCase.browser_error.push(...browser_error)
+          testCase.error.push(...error)
+          testCase.warning.push(...warning)
```

Comment:
Does it mean errors from all steps will be merged in the test case? (I do not know junit really well but it seems difficult to interpret afterwards if it is the case)

Reply:
Yes, that's what it does.
Since we're only reporting at the test level (and not the step level) we're reporting all the error at the top level.

Reply:
Ok, as long as the step tied to the error is reported, then I think it is OK.